main.py

```diff
@@ -17,15 +16,6 @@
     environ["BLISS_HOME"] = bliss_home.as_posix()
 
 
-# just returning range has issues with referencing other objects, this is more robust
-def make_range(start, stop, step=1):
-    return omegaconf.listconfig.ListConfig(list(range(start, stop, step)))
-
-
-# resolve ranges in config file
-omegaconf.OmegaConf.register_new_resolver("range", make_range, replace=True)
```

Comment:
Should this file be within the bliss directory, e.g. `bliss/main.py` or the contents should be within `bliss/api.py`? That's might hunch. We'd just have to change the pyproject.toml to point to the new location of `main()`.

Reply:
I think it's worth keeping it separate for now because I think not everything has been ported over to the new API yet, according to @zhixiangteoh. Once everything is accessible via the API it would probably be good to remove main.py.

Reply:
Sounds good, I look forward to removing main.py in a future PR